include/boost/geometry/policies/robustness/get_rescale_policy.hpp

```diff
@@ -99,10 +105,30 @@ static inline void init_rescale_policy(Geometry1 const& geometry1,
         RobustPoint& min_robust_point,
         Factor& factor)
 {
-    // Get bounding boxes
-    model::box<Point> env = geometry::return_envelope<model::box<Point> >(geometry1);
-    model::box<Point> env2 = geometry::return_envelope<model::box<Point> >(geometry2);
-    geometry::expand(env, env2);
+    // Get bounding boxes (when at least one of the geometries is not empty)
+    model::box<Point> env;
+    if (geometry::num_points(geometry1) == 0
+        && geometry::num_points(geometry2) == 0)
+    {
+        return;
+    }
+    else if (geometry::num_points(geometry1) == 0)
+    {
+        geometry::envelope(geometry2, env);
+    }
+    else if (geometry::num_points(geometry2) == 0)
+    {
+        geometry::envelope(geometry1, env);
+    }
+    else
+    {
+        geometry::envelope(geometry1, env);
+        model::box<Point> env2 = geometry::return_envelope
+            <
+                model::box<Point>
+            >(geometry2);
+        geometry::expand(env, env2);
```

Comment:
This is absolutely true: the final envelope we get this way for spherical/geographic CS may not be optimal.

What we need to do to address this issue is to overload `bg::envelope()` to take two geometries as input, and properly compute their combined envelope. This requires a lot more work though, and although I agree, I consider it as outside the scope of this PR.

We should do it though, that is be able to compute the envelope of two (or possibly more geometries), and we should put it in our TODO list as part of the on-going work for supporting spherical/geographic CS.